src/lib/jsonParseV2/parseBatchUpdate.cpp

```diff
@@ -102,9 +110,17 @@ std::string parseBatchUpdate(ConnectionInfo* ciP, BatchUpdate* burP)
     if (name == "entities")
     {
       // param 4 for parseEntityVector(): attributes are allowed in payload
-      std::string r = parseEntityVector(ciP, iter, &burP->entities, true);
+      std::string r = parseEntityVector(ciP, iter, &burP->entities, idPatternAllowed, true);
```

Comment:
`parseBatchUpdate()` never allow idPattern (at the present moment, I mean). Thus, I think a variable is not needed here, just to set the false value here. Or maybe I'm missing something...

That will save the modification in L88-103, that should be reverted to its original form.

Reply:
Fixed in 425b48c